modules/preview-react/InformationHighlight/lib/parts/Heading.tsx

```diff
@@ -0,0 +1,26 @@
+import * as React from 'react';
+import {createComponent, ExtractProps} from '@workday/canvas-kit-react/common';
+import {createStyles, handleCsProp} from '@workday/canvas-kit-styling';
+import {base, system} from '@workday/canvas-tokens-web';
+import {Heading} from '@workday/canvas-kit-react/text';
+
+const headingStyles = createStyles({
```

Comment:
also for consistency I think internally we're trying to use `createStencil` even for basic styling. This is mostly for our css transform and not external facing, so we can convert this to createStencil